setup/install/providers/aws/aws-ec2.md

```diff
@@ -102,47 +113,252 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
+
+### Create AWS IAM user
+
+1. Navigate to the AWS IAM console.
+2. Switch to the **spinnakerManaging** AWS Account.
+3. Add a user and name it **spinnakerManaging**.
+4. Check the “Programmatic access” checkbox.
+5. Add tags to help you identify this user, such as "spinnakerManaging".
+6. Create the user.
+7. Save the “Access key ID” and “Secret access key” somewhere secure, such as a secret management system like Vault, AWS Secrets manager, or 1Password.
+8. Save the AWS ARN for the user, for example: `arn:aws:iam::100000000001:user/spinnakerManaging`. You need the ARN later when you configure the managed accounts to trust the managing account (IAM user).
+
+### Create Roles
+
+#### Create Managed Roles in each target AWS Account
+
+First, create the **spinnakerManaged** role for the **develop ID=200000000002**. Then, repeat the same steps to create **spinnakerManaged** in **staging ID=300000000003**.
+
+1. Navigate to the AWS IAM console.
+2. Switch to the AWS Account you want to create the roll for.
+3. Go to **Roles > Create Role**.
+4. Select EC2. You can change this later, because we want to specify an explicit consumer of this role in a later stage.
+5. For permissions, search for "PowerUserAccess" and select this policy. This gives the role permission to access AWS services.
+6. Add tags that will help you identify this role.
+7. Enter a role name: spinnakerManaged.
+8. Create the roll.
+9. Select the created role and add an inline policy using the following JSON snippet: 
+
+   ```json
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Action": [
+                   "iam:ListServerCertificates",
+                   "iam:PassRole"
+               ],
+               "Resource": [
+                   "*"
+               ],
+               "Effect": "Allow"
+           }
+       ]
+   }
+   ```
+
+10. Name the policy "PassRole-and-Certificates".
+11. Create the policy.
+
+Repeat these steps for the second AWS environment. In this example, that is the **staging** environment.
+
+### Create the role BaseIAMRole
+
+BaseIAMRole is the default role that Spinnaker will use if the **spinnakerManaged** role is not found.
+
+1. Navigate to the AWS IAM console. 
+2. Create the role.
+3. Select **EC2**, and click **Next: Permissions**
+4. Click **Next: Tags**
+5. Add tags that will identify this Role. Spinnaker
+6. Specify the Role Name as **BaseIAMRole**
+
+Repeat this process for all the accounts.
+
+### Create AWS Policy
+
+The AWS IAM policy gives permissions to the **spinnakerManaging** user to assume the **spinnakerManaged** role for the different target accounts (**develop** and **staging**).
+
+1. Select the AWS account where Spinnaker lives **spinnakerManaging**.
+2. Access AWS IAM console.
+3. List the AWS ARN of the **spinnakerManaged** accounts **develop** and **staging**. For example, the list below shows 3 AWS ARNs for different AWS accounts, each corresponding to an environment:
```

Comment:
@aleon1220 this sentence says 3 arns, but the example only shows two. Is the sentence incorrect or is the example missing one of the arns?

Reply:
I have added it now with a bulleted list of AWS accounts for more redundancy.